sdk/include/psdk/ntddndis.h

```diff
@@ -442,4 +484,14 @@ typedef struct _NDIS_CO_LINK_SPEED {
 }
 #endif
 
+/* Full duplex driver */
```

Comment:
Preferrably move these before the terminating `#ifdef __cplusplus`, even after the last `#define NDIS_MINIPORT_64BITS_DMA ...` to get all the defines together.

Reply:
Sorry, realizing how stupid that was now. Going to resolve this and the other comment on placement now